fjord/analytics/static/css/dashboard.less

```diff
@@ -514,3 +514,7 @@ table.flagged {
     }
   }
 }
+
+.test-answer {
+   background-color: #fee !important;
```

Comment:
We try to avoid `!important` because it involves funny things. It's better to make the css selector more specific.

In this case, you can define this rule in the `.summarytable` section since we only want it to apply to the summary table anyhow. Plus that might give you enough specificity that it overrides the `tr:nth-child` rules.